jekyll/_ccie/preview.md

```diff
@@ -8,41 +8,39 @@ description: "Pre-release access to CircleCI 2.0 functionality behind your firew
 hide: true
 ---
 
-# Introducing CircleCI Enterprise 2.0
-CircleCI 2.0 is our new infrastructure that provides a number of new valuable options for your development teams, including:
+CircleCI 2.0 is new infrastructure that provides the following improvements:
```

Comment:
thanks, changed to 'CircleCI 2.0 provides new infrastructure that includes the following improvements' I'm trying to get away from using the personal 'our' and 'we' and just speak to our audience as CircleCI to avoid confusion and make it more formal.